lib/milc_interface.cpp

```diff
@@ -645,6 +645,76 @@ void qudaGaugeForce(int precision, int num_loop_types, double milc_loop_coeff[3]
   qudaGaugeForcePhased(precision, num_loop_types, milc_loop_coeff, eb3, arg, 0);
 }
 
+/**
+ * @brief Reusable routine that creates a qudaGaugeParam for gauge-related observable measurements
+ *
+ * @param [in] precision MILC precision
```

Comment:
spaces between `param` and `[in]`

Reply:
Done locally, will push soon

Reply:
https://github.com/lattice/quda/commit/4a893d83c4c5b4c15c9e0d07f8cc3cd1a0e1955a